python/tk_multi_loader/model_latestpublish.py

```diff
@@ -66,7 +66,7 @@ def get_associated_tree_view_item(self, item):
         entity_item_hash = item.data(self.ASSOCIATED_TREE_VIEW_ITEM_ROLE)
         return self._associated_items.get(entity_item_hash)
 
-    def load_data(self, item, child_folders, show_sub_items, additional_sg_filters):
+    def load_data(self, items, child_folders, show_sub_items, additional_sg_filters):
```

Comment:
i don't think changing this param will cause any issues assuming all calling code has been updated, but should definitely update the docstring to reflect.